.cirrus.yml

```diff
@@ -301,6 +301,11 @@ task:
           CHANNEL: "master"
           CHANNEL: "stable"
       << : *BUILD_ALL_PACKAGES_APP_TEMPLATE
+      create_all_packages_app_legacy_script:
+        - $PLUGIN_TOOL_COMMAND create-all-packages-app --legacy-source=.ci/legacy_project/all_packages --output-dir=legacy/ --exclude script/configs/exclude_all_packages_app.yaml
+      build_all_packages_legacy_script:
+        - cd legacy/all_packages
+        - flutter build $BUILD_ALL_ARGS --debug
```

Comment:
Why debug instead of release? Speed?

Reply:
Yes, it's arbitrary, and I assumed debug would be faster.